docs/modules/Record.ts.md

```diff
@@ -188,28 +298,66 @@ export declare function foldMap(
 export declare function foldMap<M>(M: Monoid<M>): <A>(f: (a: A) => M) => (fa: Record<string, A>) => M
 ```
 
+**Example**
+
+```ts
+import { foldMap } from 'fp-ts/Record'
+import { Ord } from 'fp-ts/string'
+import { Monoid } from 'fp-ts/Monoid'
+
+const m: Monoid<string> = { empty: '', concat: (x: string, y: string) => (x ? `${x} -> ${y}` : `${y}`) }
+const f = (a: number) => `-${a}-`
+const x = { c: 3, a: 1, b: 2 }
+assert.deepStrictEqual(foldMap(Ord)(m)(f)(x), '-1- -> -2- -> -3-')
+```
+
 Added in v2.0.0
 
 ## reduce
 
+Reduces a `Record` passing each value to the iterating function,
+sorted by key with the provided `Ord`.
+
 **Signature**
 
 ```ts
 export declare function reduce(O: Ord<string>): <A, B>(b: B, f: (b: B, a: A) => B) => (fa: Record<string, A>) => B
 export declare function reduce<A, B>(b: B, f: (b: B, a: A) => B): (fa: Record<string, A>) => B
 ```
 
+**Example**
+
+```ts
+import { reduce } from 'fp-ts/Record'
+import { Ord } from 'fp-ts/string'
+
+const x = { c: 3, a: 'foo', b: false }
+assert.deepStrictEqual(reduce(Ord)([] as string[], (b, a) => [...b, `-${a}-`])(x), ['-foo-', '-false-', '-3-'])
+```
+
 Added in v2.0.0
 
 ## reduceRight
 
+Same as `reduce` but entries are processed in reversed order.
```

Comment:
It's never explicitly said that `reduce` processes from the left. It may be beneficial to link this comment to the function's name i.e. that it processes from the right.

Reply:
ok. I'll try to clarify.